src/components/ui/creating/index.tsx

```
@@ -146,12 +146,19 @@ export default function CreatingModal({ onClose }: CreatingModalProps) {
         const files = e.target.files;
         if (files) {
             const newImages: string[] = [];
```

ESLint

[error] 'newImages' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)